website/docs/r/cloud_project_database_opensearch_user.html.markdown

```diff
@@ -86,9 +84,7 @@ The following arguments are supported:
     * `write`
     * `readwrite`
     * `deny`
-
-* `name` - (Required, Forces new resource) Username affected by this acl. A user named "avnadmin" is map with already created admin user and reset his password instead of create a new user.
-
+* `name` - (Required, Forces new resource) Username affected by this acl. A user named "avnadmin" is map with already created admin user and reset his password instead of creating a new user.
```

Comment:
```suggestion
* `name` - (Required, Forces new resource) Username affected by this acl. A user named "avnadmin" is mapped with already created admin user and reset his password instead of creating a new user.
```